### DISCUSSION TITLE
Trouble uploading proof of academic status

### DISCUSSION THREAD

**Discussion by ReeF924** [community/community#64895] [GitHub Education] [answered]
### Hi everyone,

Hello everybody I need help.
I want to apply for github campus and they want me to upload a proof of my academic status. That's fine, I have an ISIC card (international student identity card) but everytime I try to add it, github says, that my name on the card has to be the same as is in my github profile, which is! The photo is in good enough quality, I've tried multiple times already. Maybe the problem might be the fact, that I have some special letters in my name? ''á,'š','ý' to be precise. But that shouldn't cause problems, should it? I've tried to use my full name both without them and with, but neither of them worked. I've seen a similar post that said that I should translate the ID card? But how? Do I photoshop the name out and add there my name without the special characters? For instance from Lukáš to Lukas? Should I then keep in my profile name without the special characters or with them?  I have no idea what to do or what is the problem, I'd really appreciate your help, since I am out of ideas. 
Thanks in advance!

![error_Github](https://github.com/community/community/assets/127877051/0c2b0a60-fec9-46b6-8d06-45bdf153cae8)

**Comment by ghost** [comment]
I also have this problem, it seems that this problem has been around for a long time, I think this is a bug in github, but I don't know how to report the problem to the official, so that they can fix this problem as soon as possible https://github.com/orgs/community/discussions/64847

**Comment by arafajs** [comment]
GitHub may not be able to recognize them as letters, and that is why it is saying that the name on the card does not match the name in your profile.

You could try translating the ID card, but I am not sure how you would do that. 

I think the best thing to do would be to contact GitHub Education Support and ask them for help. They may be able to tell you what the problem is and how you can fix it.

**Reply by ghost** [reply]
I saw that many people encountered this problem in the community. The official has not responded to how to solve this problem. I also sent an email to the official, but there is no reply

**Reply by arafajs** [reply]
Yes. I think this is a known issue.
You can wait for a fix from GitHub. They are aware of this issue and are working on a solution.

**Reply by ghost** [reply]
are you sure? I sent an email and no reply yet.

**Reply by arafajs** [reply]
Yes. Github is sensitive to special characters even in repo names. So it's a common problem.

**Reply by ghost** [reply]
You said that I also encountered this problem. What I said should be another question, you can look at the question number 64847, my question is mainly this https://github.com/orgs/community/discussions/64847

**Comment by ReeF924** [comment, marked as answer]
Hello, I've managed to solve the problem, I've just used photoshop and just replaced the special letters (from š to s, á to a etc.) and I've done the same on my github profile, it worked and I've been approved. For those who cannot replace letters like this easily I can't help unfortunately. Thanks for your help guys.

**Comment by SnowPants** [comment]
hi there isn't a way to upload academic status paperwork even thought it states on the page there is a button to upload? anyone have any clue or resolution to this problem?
